tests/commands/module.py

```diff
@@ -77,9 +77,13 @@ def runTest(self):
 
 class F41_TestCase(F31_TestCase):
     def runTest(self):
-        # F41 warns about deprecation
-        with self.assertWarns(KickstartDeprecationWarning):
-            F31_TestCase.runTest(self)
+        # make sure we've been deprecated
+        parser = self.getParser("module")
+        self.assertEqual(issubclass(parser.__class__, DeprecatedCommand), True)
```

Comment:
You need to import `DeprecatedCommand` and drop the `KickstartDeprecationWarning` import.